pkg/validation/commands_test.go

```diff
@@ -144,6 +149,26 @@ func TestValidateCommands(t *testing.T) {
 				generateDummyCompositeCommand("composite-a", []string{"basic-exec"}, nil),
 			},
 		},
+		{
+			name: "Invalid command with import source attribute",
+			commands: []v1alpha2.Command{
```

Comment:
would probably update one of the generate test funcs to take attr, and possibly if you're updating it, for the other tests too

Reply:
those added unit tests for commands/components/projects/startprojects are taking attributes with import&override information
 Can you explain what you are expecting?

Reply:
`generateDummyApplyCommand(name, component string, group *v1alpha2.CommandGroup, attributes attributesPkg.attributes) ` is what i was suggesting

Reply:
updated